esp/src/src-react/components/DFSCheck.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+
```

Comment:
@kunalaswani I don't recall if this was what the GH actions was complaining about last time, but you'll need to delete the extra blank lines. You can also check for this kind of stuff by executing `npm run lint`. I probably forget to do it before opening a PR almost half the time.

Reply:
I'll keep that in mind, thank you, i thought i had removed those on the last update, sorry.